### PR TITLE
Issue/msg contains

### LINKED ISSUE #443: Message log filter does not take list as parameter for msg_contains
**Describe the bug**
The function **get_message_log_entries** excepts Iterable as variable. But if the example below is used no entries are returned. If only one of them is used (depending on the language of the server) entries are found. 

**To Reproduce**
```python
with TM1Service(**TM1_PARAMETERS) as tm1:
    messagelog = tm1.server.get_message_log_entries(reverse=False, logger= 'TM1.Server' , msg_contains= ['TM1 Server is ready','TM1 Server ist bereit,'])
```

**Expected behavior**
Entries should show up if only on of the entries in the list is valid

**Version**
Current GitHub Master
TM1 Build Number:  11.8.00200.24

### PR DESCRIPTION
**Pull request by MariusWirtz**
Fixes #443

### CODE REVIEW

File: TM1py/Services/ServerService.py [hunk 1]
```diff
@@ -87,10 +88,15 @@ def get_message_log_entries(self, reverse: bool = True, since: datetime = None,
         :param logger: string, eg TM1.Server, TM1.Chore, TM1.Mdx.Interface, TM1.Process
         :param level: string, ERROR, WARNING, INFO, DEBUG, UNKNOWN
         :param msg_contains: iterable, find substring in log message; list of substrings will be queried as AND statement
+        :param msg_contains_operator: 'and' or 'or'
 
         :param kwargs:
         :return: Dict of server log
         """
+        msg_contains_operator = msg_contains_operator.strip().lower()
+        if msg_contains_operator not in ("and", "or"):
```
Comment (rkvinoth) [inline TM1py/Services/ServerService.py:97]: Can we make this case insensitive?

Comment (MariusWirtz) [reply TM1py/Services/ServerService.py:97]: it is :)

Comment (rkvinoth) [reply TM1py/Services/ServerService.py:97]: haha didn't notice the previous line :D

Comment (rkvinoth) [reply TM1py/Services/ServerService.py:97]: looks good from my end.

Comment (MariusWirtz) [reply TM1py/Services/ServerService.py:97]: Thanks

**Pull request merged**